src/utils/eventsBinder.js

```diff
@@ -1,12 +1,13 @@
 /* vim: set softtabstop=2 shiftwidth=2 expandtab : */
 
-import _ from 'lodash'
+import _ from 'lodash';
 
 export default (vueElement, googleMapObject, events) => {
   _.forEach(events, (eventName) => {
-    const exposedName = eventName;
+    const exposedName = 'g-' + eventName;
```

Comment:
Sorry to try again, but you really want change this idea, I feel it's more readable.